Makefile.rules

```diff
@@ -446,8 +446,9 @@ etags:
 # Rules for 'make clean'
 ########################################################################
 distclean: clean
-	rm -f config.log config.status libtool GNUmakefile configure
+	rm -f config.log config.status libtool GNUmakefile
 	rm -f doc/make_doc
+	rm -f doc/*/*.aux doc/*/*.bbl doc/*/*.blg doc/*/*.brf doc/*/*.idx doc/*/*.ilg doc/*/*.ind doc/*/*.log doc/*/*.out doc/*/*.pnr doc/*/*.tex
```

Comment:
`doc/*/*.toc` are also generated by LaTeX and can be removed.

Reply:
Thanks, I removed .toc (not sure how I lost that, as the code where I copied the list from had it. Huh)